### PR TITLE
feat: show recent past events

### LINKED ISSUE #1705: Add Previous events to event page
### Description

We would like to display previous events in the event module.
- It makes the page more rich
- It could help people to find eachother locally even after events.

### Page related

Will be used in : [Event page](https://community.preciousplastic.com/events)

### Build suggestion
- Previous events should be greyed out. 
- They are loaded with infinite scroll as the [how-to](https://community.preciousplastic.com/how-to) page)

### Mockup
events-previous

### PR DESCRIPTION
**Pull request by Xyli0**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

Displaying a list of the 20 last (chronologically) events . The opacity of these list items is reduced compared to the upcoming events list.

## Git Issues

Closes #1705 

## Screenshots/Videos

**Data found**

Screenshot 2022-06-11 at 21-21-21 Community Platform

**Upcoming or past events not found**

Screenshot 2022-06-11 at 21-25-07 Community Platform

These are two different states on one screenshot.

1. Upcoming events are not found:
  Title disappears.
   In place "At the moment there is no upcoming events" is shown with a button to create a new event below.

3. Past events are not found:
  Title stays in place.
  "Event not found" is displayed

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of a monthly dev call (first Monday of the month, open to all!).

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on slack in the `platform-dev` channel.

### CODE REVIEW

**Comment by Xyli0** [comment]
Hello everyone,
This is my first pull request for this project (and one of the first I've done).
I want to thank the maintainers for being so friendly to a newcomer/newbie like me. 
I hope my contribution will help this wonderful project and I look to help more in the future.

Thanks!

**Comment by davehakkens** [comment]
Nice work @Xyli0. Looks/works good here!
Only remark I would have is that it would be nice if the heading text `"previous events"` Would have the same font-size as the one above `"Precious Plastic events from around the world"`

**Comment by Xyli0** [comment]
Titles are now the same size as suggested by @davehakkens.

File: src/pages/Events/EventCard/EventCard.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { useTheme } from '@emotion/react'
 
 interface IProps {
   event: IEvent
+  inactive?: boolean
```
Comment (thisislawatts) [inline src/pages/Events/EventCard/EventCard.tsx:12]: **suggestion:** Ideally we could use `isPastEvent` here to communicate the meaning. This is easier to read and better relates to the domain. An Event being in an Active or Inactive state may not be clear to new contributors coming to the codebase.

Reference: [Java Coding Standard](https://www.comp.nus.edu.sg/~cs2103/AY1617S1/contents/coding-standards-java.html#:~:text=Boolean%20variables%20should%20be%20prefixed%20with%20'is'&text=This%20is%20the%20naming%20convention,names%20like%20status%20or%20flag.), the reasoning here is sound even though it is the coding standard from a different language 😉

**Review by thisislawatts — CHANGES_REQUESTED** [review]
Thanks for raising this PR @Xyli0 ✨ Much appreciated, This is a really good quality PR!

I have a few comments but hopefully they all make sense. Please let me know if you have any further questions on them, happy to discuss further.

In case you are curious about the **bold** prefixes in my comments, I use https://conventionalcomments.org as a guide for structuring my feedback.

**Comment by Xyli0** [comment]
Thanks @thisislawatts for the feedback and the useful resources.
I agree with all the changes you mentioned. 
I will apply them and comeback with a new version. 
Don't pay attention to this last commit. I mistakenly fetched upstream on my fork 🤭

**Comment by Xyli0** [comment]
For the record I will do another PR with the aforementioned changed implemented because I messed up when fetching upstream...

**Comment by thisislawatts** [comment]
Thanks for the update @Xyli0, let me know if there's anything I can help out with :)

**Pull request closed without merging**